rewrite-java-core/src/main/resources/META-INF/rewrite/rewrite.yml

```diff
@@ -66,4 +134,27 @@ recipeList:
 
   - com.azure.recipes.core.v2.ContextRecipe
   - com.azure.recipes.core.v2.RetryOptionsConstructorRecipe
-  - com.azure.recipes.core.v2.TypeReferenceRecipe
\ No newline at end of file
+  - com.azure.recipes.core.v2.TypeReferenceRecipe
+  - com.azure.recipes.core.v2.HttpLogOptionsRecipe
+  - com.azure.recipes.core.v2.RemoveFixedDelayRecipe
+
+  # Recipe to re-order imports to match standard conventions
+  - org.openrewrite.java.OrderImports
+
+  # ----------------------------------
+  #
+  # Text-Based Replacement Recipes
+  #
+  # ----------------------------------
+
+  # Recipe to replace retryOptions argument in httpRetryOptions HttpTrait method
+  # to httpRetryOptions
+  - org.openrewrite.text.FindAndReplace:
+      find: 'httpRetryOptions(HttpRetryOptions retryOptions)'
+      replace: 'httpRetryOptions(HttpRetryOptions httpRetryOptions)'
+
+  # Recipe to replace HttpLogDetailLevel call with HttpLogOptions.HttpLogDetailLevel
+  - org.openrewrite.text.FindAndReplace:
+      find: '(?<!\.)[\w]*[\s]*HttpLogDetailLevel'
+      replace: 'HttpLogOptions.HttpLogDetailLevel'
```

Comment:
Need to handle for cases where the HttpLogDetailLevel could be qualified with HttpLogOptions.